Procfile

```diff
@@ -1,2 +1,3 @@
 tsc: npm run watch:tsc
-webpack: npm run watch:webpack
```

Comment:
procfile is used only by foreman?

Reply:
Yep, it allows to run multiple shell commands from one terminal.  
It allows to have multiple watches to concurrently run